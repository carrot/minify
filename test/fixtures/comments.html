<p>wow</p>
<!-- a comment -->
<p>omg</p>
