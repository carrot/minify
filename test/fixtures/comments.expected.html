<p>wow</p><p>omg</p>
